test/mocha/unit/handlers/router-entities/route-caching/dynamo-route-caching-provider.test.ts

```diff
@@ -116,6 +120,20 @@ describe('DynamoRouteCachingProvider', async () => {
   setupTables(TEST_ROUTE_TABLE)
   const dynamoRouteCache = new DynamoRouteCachingProvider({ cachedRoutesTableName: TEST_ROUTE_TABLE.TableName })
 
+  it('Generates cached route db entry properly with ttl based on chain id and blocks to live', async () => {
```

Comment:
Can we write a test that actually compares the DbEntry arguments? :3

Reply:
done :D